docs/modules/integrations/pages/xray-exporter.adoc

```diff
@@ -63,6 +61,47 @@ xray-exporter.project-key=ABBA
 xray-exporter.json-results-directory=/Users/happytester/Repositories/app-tests/output/results/jbehave
 ----
 
+=== Test Execution
+
+[cols="2,1,3", options="header"]
+|===
+
+|Property
+|Required
+|Description
+
+|`xray-exporter.test-execution-key`
+|false
+|The key of a `Test Execution` to which the exported test cases along with their statuses will be added
+
+|`xray-exporter.test-execution-summary`
+|false
+|The `Test Execution` summary
+
+|===
+
+Test execution import varies depending on values in `xray-exporter.test-execution-key` and `xray-exporter.test-execution-summary`, the following matrix shows this behavior change:
+
+[cols="1,1", options="header"]
+|===
+
+|Configuration
+|Result
+
+|Both `xray-exporter.test-execution-key` and `xray-exporter.test-execution-summary` are set
+|Test execution summary and related test cases will be updated
```

Comment:
not clear
- what is `related`?
- what exactly is going to be updated in test cases?